packages/formation-react/src/components/Telephone/Telephone.unit.spec.jsx

```diff
@@ -171,6 +171,20 @@ describe('Widget <Telephone />', () => {
     wrapper.unmount();
   });
 
+  // notClickable
+  it('should render a span instead of an a tag', () => {
+    const wrapper = shallow(
+      <Telephone contact={CONTACTS.GI_BILL} className="foo" notClickable />,
+    );
+    const props = wrapper.props();
+    expect(wrapper.exists('span')).to.equal(true);
+    expect(wrapper.exists('a')).to.equal(false);
+    expect(props.href).to.not.exist;
+    expect(props.className).to.equal('no-wrap foo');
```

Comment:
This assertion is failing in [Jenkins](http://jenkins.vfs.va.gov/blue/organizations/jenkins/testing%2Fveteran-facing-services-tools/detail/sanlouise-add-nonclickable-telephone/8/pipeline). It makes sense that it is failing because `foo` is being passed as the `className` prop, but the assertion is expecting to see `no-wrap foo`. 

This assertion may not even be necessary because it seems to be testing that props are passed correctly, which is something that we can reasonably trust React to do consistently.